raiden/api/python.py

```diff
@@ -246,7 +247,8 @@ def get_channel(
         if not is_binary_address(partner_address):
             raise InvalidBinaryAddress("Expected binary address format for partner in get_channel")
 
-        channel_list = self.get_channel_list(registry_address, token_address, partner_address)
+        with opentracing.tracer.start_span("get_channel"):
+            channel_list = self.get_channel_list(registry_address, token_address, partner_address)
```

Comment:
The span name doesn't seem to match with the operation that it wraps?
The same with the next two.